package.json

```diff
@@ -70,7 +70,7 @@
   },
   "dependencies": {
     "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client#PP-357/dapp-update",
-    "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#feature/PP-278/PP-329/upgrade_contracts",
+    "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#PP-357/dapp-update",
```

Comment:
Need to be updated once we merge the related PRs

Reply:
updated